modules/gitops-creating-a-new-client-in-dex.adoc

```diff
@@ -0,0 +1,31 @@
+// Module is included in the following assemblies:
+//
+// * configuring-sso-for-argo-cd-on-openshift
+
+= Enabling the Dex openshift OAuth Connector
+
+Dex uses the users and groups defined within OpenShift by querying the platform provided `OAuth` server.
```

Comment:
I'm not exactly sure what `querying` is supposed to convey here. This is how I understand the sentence, am I right? 

`Dex uses the users and groups defined within OpenShift and confirms these users and groups by checking the platform provided OAuth server.`

If I am right with this understanding, please consider a similar rephrase to be clearer.

Reply:
@jc-berger not sure about the confirm part since in the developer docs it is not mentioned. But changed "querying" to "checking"